packages/contracts/contracts/core/component/Permissions.sol

```diff
@@ -6,10 +6,15 @@ pragma solidity 0.8.10;
 
 import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
 import "@opengsn/contracts/src/BaseRelayRecipient.sol";
+import "@opengsn/contracts/src/interfaces/IRelayRecipient.sol";
 
 import "./../IDAO.sol";
 import "./../acl/ACL.sol";
 
+interface Relay {
+    function trustedForwarder() external view returns (address);
+}
+
```

Comment:
What is this interface needed for? Can we add NatSpec descriptions?

Reply:
it is needed so that solidity can call that function right away without on the DAO.sol. We can not add trustedForwarder on IDAO because it should not be part of IDAO, so just to make the compiler silent, we cast the address to Relay type to be able to call it.

Reply:
Ah, ok. Maybe a brief NatSpec dev comment would be helpful.